platforms/kubernetes/actions/deployments/rollout.ts

```diff
@@ -31,8 +113,10 @@ export default async function rollout(
       revisionName,
       namespace: Namespace.forSite(site),
     }),
-    "serving.knative.dev",
-    "v1",
-    "routes",
+    GROUP_SERVINGKNATIVEDEV,
+    VERSION_V1,
+    PLURAL_ROUTES
   );
+
+  await updtadeOldRevision({ oldRevision, site, ctx });
```

Comment:
We have a choice here.

1 - Or you should fail when you can't get the previous prod revision (then you can use `await` here)
2 - Or you don't care about that and you don't need to `await` here. It seems to be that we have a mixed solution, which you catch errors - and ignore them, but here you're "await" it. Let's simplify. how about not `await` here?